datafusion/common/src/config.rs

```diff
@@ -470,7 +478,7 @@ config_namespace! {
         /// delta_byte_array, rle_dictionary, and byte_stream_split.
         /// These values are not case sensitive. If NULL, uses
```

Comment:
> These values are not case sensitive.

this suggests the code reading this value does normalization and that it's no longer needed, can be simplified

Reply:
I actually think this is a good info for users that you assign any values to it - "plain", "PLAIN", "Plain", etc. Also, I think that's what Oleks asked in https://github.com/apache/datafusion/pull/13576#discussion_r1880549113 - so maybe let's keep it as is?